tests/unit/amo/pages/TestUserProfileEdit.js

```diff
@@ -3,13 +3,16 @@ import { createMemoryHistory } from 'history';
 import userEvent from '@testing-library/user-event';
 import { createEvent, fireEvent, waitFor } from '@testing-library/react';
 
+import * as constants from 'amo/constants';
```

Comment:
lets import the specific constants here too - there won't be that many.

Reply:
I've tried to mock isMzaBranding without importing the entire module but nothing seemed to work. I can put it in another file, but the only way i've gotten it to mock properly is with ```jest.spyOn(constants, 'isMzaBranding').mockReturnValue(true);``` where constants is the module.